### DISCUSSION TITLE
Sites ?

### DISCUSSION THREAD

**Discussion by Sleepy-Git-User** [redraskal/r6-dissect#22] [Q&A] [answered]
Hello, I was wondering if your able to see which site was played. As when reviewing the json example I couldnt seem to find it. Thanks.

**Comment by redraskal** [comment]
@Sleepy-Git-User Not at the current time, this is on my todo list to implement

**Reply by Sleepy-Git-User** [reply]
Ah okay. Thats great to hear. Do you have a rough timeline for when that feature will be implemented?

**Reply by redraskal** [reply]
@Sleepy-Git-User Almost finished, likely tonight

**Reply by Sleepy-Git-User** [reply]
Thats actually brilliant to hear. Thanks!

**Comment by redraskal** [comment, marked as answer]
Feature is released on `main` branch

**Reply by Sleepy-Git-User** [reply]
[test json.txt](https://github.com/redraskal/r6-dissect/files/10914318/test.json.txt)
I tested the main branch and it doesnt appear to list sites.

**Reply by redraskal** [reply]
It should work on recent replays, looks like this logic is different in older versions

**Reply by Sleepy-Git-User** [reply]
Hello again, after checking both SI replays and Rank replays from Year8.1 sites still dont appear to be present.

**Reply by redraskal** [reply]
Working on #26, it looks like role swapping causes problems

**Reply by Sleepy-Git-User** [reply]
Ahh okay. Has adding sites been added back to the todo list? If so ill keep an eye on there for when its fixed. Thanks!
